### ISSUE TITLE
Find a way to read token.json without temporarily copying it into the current directory

### ISSUE THREAD

**Issue by utkuufuk** [utkuufuk/google-budget#2]
`file.Storage()` function does not accept relative/absolute paths other than the current directory. Must find a way to read `token.json` from different directory.

**Comment by utkuufuk** [comment]
Solved by 02affb7c37269a78189aa7ff0c57e6681086fe0f

Working directory is now temporarily changed to read token.json and authorize.

**Comment by adammartiniello** [comment]
Hello @utkuufuk 

I am attempting to install this app but getting the following errors when running the install.sh. Would you be able to help?

```
~/.local/lib/python3.7/site-packages/oauth2client/_helpers.py:255: UserWarning: Cannot access token.json: No such file or directory
  warnings.warn(_MISSING_FILE_MESSAGE.format(filename))
Traceback (most recent call last):
  File "~/.local/lib/python3.7/site-packages/oauth2client/clientsecrets.py", line 86, in _validate_clientsecrets
    (client_type, client_info), = clientsecrets_dict.items()
ValueError: too many values to unpack (expected 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "createtoken.py", line 9, in <module>
    flow = client.flow_from_clientsecrets('credentials.json', scopes)
  File "~/.local/lib/python3.7/site-packages/oauth2client/_helpers.py", line 133, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "~/.local/lib/python3.7/site-packages/oauth2client/client.py", line 2135, in flow_from_clientsecrets
    cache=cache)
  File "~/.local/lib/python3.7/site-packages/oauth2client/clientsecrets.py", line 165, in loadfile
    return _loadfile(filename)
  File "~/.local/lib/python3.7/site-packages/oauth2client/clientsecrets.py", line 126, in _loadfile
    return _validate_clientsecrets(obj)
  File "~/.local/lib/python3.7/site-packages/oauth2client/clientsecrets.py", line 89, in _validate_clientsecrets
    _INVALID_FILE_FORMAT_MSG + ' '
oauth2client.clientsecrets.InvalidClientSecretsError: Invalid file format. See https://developers.google.com/api-client-library/python/guide/aaa_client_secrets Expected a JSON object with a single property for a "web" or "installed" application
```
